docs/visualize/controls.asciidoc

```diff
@@ -0,0 +1,61 @@
+[[controls]]
+== Controls Visualization
+
+Controls provide the ability to add interactive inputs to Kibana Dashboards.
+Controls can be used to create two types of inputs; dropdowns and range sliders.
+
+image::images/controls/controls_in_dashboard.png[]
+
+=== Configuration
+Create Controls visualizations by following the <<createvis,Creating a Visualization>> instructions.
+Select *Controls* as the visualization type.
+
+==== Dropdown
+
+Dropdown inputs allow users to easily filter content by selecting options from a list.
+The dropdown options list is dynamically populated with the results of a terms aggregation.
+
+Add dropdown inputs to your Controls Visualization by selecting *Options list* and clicking *Add*.
+
+image::images/controls/dropdown_control_editor.png[]
+
+*Control Label*:: By default, the control label will be the *Field*. Set this configuration to customize the control's label.
+*Index Pattern*:: Specify the data set by selecting an <<index-patterns,Index Pattern>>.
+*Field*:: Select the Field that will be used to populate the options list
+and will be filtered on when users interact with the input.
+The list of available fields is derived from the selected *Index Pattern* configuration.
+*Parent control*:: Set parent control to chain dropdown inputs so that the selection of the first input
+will filter the terms of the second input. Configuration is only available when multiple dropdowns have been created.
```

Comment:
> when multiple dropdowns have been

I think it's also available when range is added.  So maybe say "when multiple inputs have been created".  A little confusing because "input" can mean an input control, or the input field inside an input control.  Not sure how to distinguish though...